File: targets/numworks/tests/camlboot/benchs/fibo.py
from time import *

def fibo(n):
    if n < 2:
        return n
    else:
        return (fibo(n-1)+fibo(n-2))

def test_fibo(n):
    i = 0
    while (i< n):
        fibo(5);
        i = i + 1

# display.show(Image.CONFUSED)
start = monotonic()
test_fibo(1000)
stop = monotonic()
print(stop-start)
# display.show(Image.HAPPY)

File: targets/numworks/tests/camlboot/benchs/fibo.ml
let rec fibo n =
  if n <= 1 then n
  else (fibo (n-1) + fibo (n-2));;

let repeat n =
  for i = 0 to n do
    let _ = fibo 5 in ()
  done

let _ =
  let start = millis () in
  repeat 1000; (* x 1000 to compare with OMicroB *)
  let stop = millis () in
  print_int (stop-start)
